src/ui/Badge/index.tsx

```diff
@@ -1,15 +1,20 @@
-import React, { useContext } from 'react';
-import PropTypes from 'prop-types';
+import React, { ReactElement, useContext } from 'react';
 
 import './index.scss';
 import Label, { LabelTypography, LabelColors } from '../Label';
 import { LocalizationContext } from '../../lib/LocalizationContext';
 
+export interface BadgeProps {
+  count: number;
+  maxLevel?: number;
+  className?: string | Array<string>;
+}
```

Comment:
is, className?: string | string [], in script_d_ts
Want to keep them same?

Reply:
let me keep them same